### PR TITLE
privatezilla: Add version 0.43.0

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/privatezilla.json
Comment (Ash258) [inline bucket/privatezilla.json:18]: Why do you use this instead of normal github chekver?

File: bucket/privatezilla.json
Comment (Ash258) [inline bucket/privatezilla.json:15]: Does not make sense to persist this folder, when there is no merge support. Persist and merge it manually or remove it.

File: bucket/privatezilla.json
Comment (Ash258) [inline bucket/privatezilla.json:8]: https://github.com/builtbybel/privatezilla/blob/0c24b54027311ddec8bf81aa6069b7b2da8809f9/src/Privatezilla/Program.cs#L20

```suggestion
```

File: bucket/privatezilla.json
Comment (Ash258) [inline bucket/privatezilla.json:3]: > Security and privacy checker for Windows 10

File: bucket/privatezilla.json
Comment (Ash258) [inline bucket/privatezilla.json:8]: Add pre_install check for windows version https://github.com/builtbybel/privatezilla#system-requirements

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/privatezilla.json
Comment (linsui) [reply bucket/privatezilla.json:18]: Looks like this is the offcial update checker. No special reasons.

**Review by rasa — APPROVED** [review]

**Comment by Ash258** [comment]
@rasa Why did you merge this??

**Comment by rasa** [comment]
Before the requested changes appear to have been made, it passed review, and many are asking if this project is dead, as merging is not occurring with the frequency of the past. If you would prefer I defer to you on merge requests, I would happy to do so.

**Comment by Ash258** [comment]
They are asking that in Main, not here.

I am always merging on Sunday evening as soon as there are some approved requests, which I full tested. So i have always a week per PR for extensive verification. Over week I am doing manifest format review and occasionally functionality tests. On weekends I am reading through documentation and extensive testing.

Merge here anytime, but **only** the approved PRs.

**Pull request merged**
